scripts/authors.ts

```diff
@@ -1,16 +1,18 @@
-import fs = require('fs');
-import path = require('path');
-import child_process = require("child_process");
+import fs = require("fs");
+import path = require("path");
+import childProcess = require("child_process");
 
-type Author = {
+interface Author {
     displayNames: string[];
     preferredName?: string;
     emails: string[];
 };

```

Comment:
When changing a type alias to an interface, we should remove the trailing `;` (here and on lines 13 and 18 are immediately obvious, but I'll keep looking for other instances).